### PR TITLE
load image from numpy array

### PR DESCRIPTION
**Pull request by DeusAnimo**

### CODE REVIEW

File: lunavl/sdk/image_utils/image.py [hunk 1]
```diff
@@ -167,13 +167,21 @@ def __init__(
             error = self.coreImage.loadFromMemory(body, len(body), imgFormat)
             if error.isError:
                 raise LunaSDKException(LunaVLError.fromSDKError(error))
+        elif isinstance(body, np.ndarray):
+            arr = body.__array_interface__
+            try:
+                typekey = (1, 1) + arr["shape"][2:], arr["typestr"]
```
Comment (gluhar2006) [inline lunavl/sdk/image_utils/image.py]: is it possible to make it clearer?

File: lunavl/sdk/image_utils/image.py [hunk 2]
```diff
@@ -167,13 +167,21 @@ def __init__(
             error = self.coreImage.loadFromMemory(body, len(body), imgFormat)
             if error.isError:
                 raise LunaSDKException(LunaVLError.fromSDKError(error))
+        elif isinstance(body, np.ndarray):
+            arr = body.__array_interface__
```
Comment (gluhar2006) [inline lunavl/sdk/image_utils/image.py]: is there another way to do this without accessing to private field?

**Comment by gluhar2006** [comment]
test(s)

File: lunavl/sdk/image_utils/image.py (hunk 2 above)
Comment (DeusAnimo) [reply lunavl/sdk/image_utils/image.py]: fixed

File: lunavl/sdk/image_utils/image.py (hunk 1 above)
Comment (DeusAnimo) [reply lunavl/sdk/image_utils/image.py]: well, I've done the best I can

File: lunavl/sdk/image_utils/image.py [hunk 3]
```diff
@@ -167,13 +167,21 @@ def __init__(
             error = self.coreImage.loadFromMemory(body, len(body), imgFormat)
             if error.isError:
                 raise LunaSDKException(LunaVLError.fromSDKError(error))
+        elif isinstance(body, np.ndarray):
```
Comment (gluhar2006) [inline lunavl/sdk/image_utils/image.py:170]: what about test(s)?

**Pull request merged**